qt_binder/tests/test_binder.py

```diff
@@ -59,7 +59,11 @@ def test_object_traits(self):
                   for k, v in self.Object.class_traits().items()}
         self.assertIsInstance(traits['deleteLater'], QtSlot)
         self.assertIsInstance(traits['destroyed'], QtSignal)
-        self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        try:
+            self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        except KeyError:
+            # on Python 3, key name is different
+            self.assertIsInstance(traits["destroyed_b'QObject*'"], QtSignal)
```

Comment:
This behavior is a bug, so it needs to be fixed in the code (probably by [`.decode()`ing instead of `str()`ing](https://github.com/enthought/qt_binder/blob/master/qt_binder/binder.py#L556)), rather than changing the test. What version of PyQt4 are you using? I'm not able to set up a new testing environment at the moment due to a Sourceforge outage.